src/TorchSharp/Torch.cs

```diff
@@ -304,26 +304,82 @@ public static partial class nn
             public static partial class utils
             {
                 [DllImport("LibTorchSharp")]
-                extern static double THSTensor_clip_grad_norm_(IntPtr tensor, int len, double max_norm, double norm_type);
+                extern static double THSTensor_clip_grad_norm_(IntPtr tensors, int len, double max_norm, double norm_type);
+
+                [DllImport("LibTorchSharp")]
```

Comment:
nit: for the sake of coding style, could you please decorate these methods with "private" specifiers? It is always better to be explicit about that.